trustpinning/certs.go

```diff
@@ -161,6 +161,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun string, trus
 		logrus.Debugf("failed to verify TUF data for: %s, %v", gun, err)
 		return nil, &ErrValidationFail{Reason: "failed to validate integrity of roots"}
 	}
+	signedRoot.Signatures = root.Signatures
```

Comment:
I think root is the only role that has this weird case, but @avaid96 can confirm